src/display_list/mod.rs

```diff
@@ -32,7 +32,7 @@
 //!
 //! The above snippet has been built out of the following structure:
 //!
-//! ```
+//! ```rust,ignore
```

Comment:
A leftover or do you think this doc example doesn't make sense?

Reply:
Is leftover